api-client/src/Endpoint.ts

```diff
@@ -61,7 +61,9 @@ class Endpoint<P, T> {
     const response = await fetch(url, requestOptions).catch((e: Error) => {
       throw new FetchError({
         endpointName: this.stateName,
-        innerError: e
+        innerError: e,
+        url,
```

Comment:
Can you explain why it is necessary to add this params or how they fix the error?

Reply:
Just adds more information to debug, not related to fixing the actual error.